content/en/core/overview/architecture.md

```diff
@@ -8,7 +8,80 @@ description: >
 
 ## Overview
 
-![Architecture of a CHT project](architecture.png)
+```mermaid
```

Comment:
I think the replacement diagram is aesthetically unappealing and we break this out into it's own PR so we can involve UX/design folks before pushing.  Hopefully we'll get there, but having the odd `.png` where we need pixel perfect art doesn't negate the ease of a quick mermaid diagram!